cfme/configure/settings.py

```diff
@@ -23,109 +25,173 @@ class TimeProfileAddForm(View):
     save_button = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
     configuration = Dropdown('Configuration')
     table = Table("//div[@id='main_div']//table")
-    save_edit_button = Button('Save')
     cancel_button = Button('Cancel')
     help_block = Text("//span[contains(@class, 'help-block')]")
 
 
-class TimeprofileAddEntities(View):
+class TimeProfileAddEntities(View):
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h3')
 
 
 class TimeProfileAddFormView(BaseLoggedInPage):
-    timeprofile_form = View.nested(TimeProfileAddForm)
-    entities = View.nested(TimeprofileAddEntities)
+    time_profile_form = View.nested(TimeProfileAddForm)
+    entities = View.nested(TimeProfileAddEntities)
     mysetting = View.nested(MySettingsView)
 
 
-class Timeprofile(Updateable, Navigatable):
-    def __init__(self, description=None, scope=None, days=None, hours=None, timezone=None,
-                 appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.description = description
-        self.scope = scope
-        self.days = days
-        self.hours = hours
-        self.timezone = timezone
+class TimeProfileEditView(TimeProfileAddFormView):
+    save_edit_button = Button('Save')
```

Comment:
Please use `save` here for the widget name. We're trying to convert uses of `<action>_button` to just `<action>` names.